package.json

```diff
@@ -5,7 +5,7 @@
     "Jeff Smick"
   ],
   "description": "libxml bindings for v8 javascript engine",
-  "version": "0.14.0",
+  "version": "0.14.1",
```

Comment:
Tip: don't do this in PRs. It will never be merged because it is up to the maintainer to decide what the next version will be.